src/pages/listing/listing.css

```diff
@@ -0,0 +1,337 @@
+@value pure: "../../_pure.css";
```

Comment:
This file has lots of extra linebreaks, normalize it to one blank line between rules